cytoscape-expand-collapse.js

```diff
@@ -358,15 +355,16 @@ module.exports = function (params, cy, api, $) {
     }
   };
 
+	functions.unbind.apply(cy.container());
```

Comment:
This line doesn't exist in cueUtilities.js file, but somehow emerged here and causes error. Do we need this line?

Reply:
@hasanbalci Should be fixed after rebuilt...